server/controllers/entries.js

```diff
@@ -1,133 +1,197 @@
-import data from '../models/data';
+import bcrypt from 'bcrypt';
+import dotenv from 'dotenv';
+import auth from '../helpers/authentication';
+import db from '../models/index';
+import { addEntry } from '../models/model.queries';
+
+dotenv.config();
+
 
-const entry = data;
 /**
  *
  *@description - Class Definition for the Entry class
  *
  * @export
  *
- * @class Entry
+ * @class Entries
  */
-class Entry {
+class Entries {
   /**
-   *@description - Fetch all entries
-  *@param {object} request - request object
-   *
-   * @param {object} response - responce object
+  *@description - Create an Enrty
    *
-   * @return {object} return object as response
-   *
-   * @memberof Entry
-   * */
-  static getAll(request, response) {
-    return response.status(200).send({
-      message: 'Successful',
-      entry: data,
-      error: false
-    });
-  }
+  *@param {object} request - request object
+  *
+  * @param {object} response - response object
+  *
+  * @return {object} return object as response
+  *
+  * @memberof Entries
+  */
+  static createEntry(request, response) {
+    const {
+      title, mood, entry
+    } = request.body;
+    console.log(request.decoded);
+    db.query(addEntry(title, mood, entry, request.decoded.id))
+      .then((result) => {
+        response.status(201).send({
+          message: 'Entry Created Successfully',
+          newRequest: {
+            id: result.rows[0].id,
+            userId: request.decoded.id,
+            title,
+            mood,
+            entry,
+            status: 'Successful'
+          }
 
-  /**
- *@description - Fetch one entry
- *@param {object} request - request object
- *
- * @param {object} response - responce object
- *
- * @return {object} return object as response
- *
- * @memberof Entry
-*/
-  static getOne(request, response) {
-    const index = entry.findIndex(item => item.entryId === parseInt(request.params.entryId, 10)); {
-      if (!index) {
-        return response.send({
-          message: 'Successful',
-          entry: data[index],
-          error: false
         });
-      }
-    }
-    return response.status(404).send({
-      message: 'Entry not found!',
-      error: true
-    });
+      })
+      .catch((error) => {
+        response.status(500).send({
+          message: 'Create Entry Failed',
+          error: error.message,
+          status: 'fail'
+        });
+      });
   }
+// /**
```

Comment:
what's the reason for this commented code?

Reply:
those are the endpoints i'm modifying, thats why i commented them out.